.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+on: push
+name: CI
+jobs:
+  fmt:
+    name: Rustfmt
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - run: rustup component add rustfmt
+      - uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+          args: --all -- --check
```

Comment:
```suggestion
          args: --all -- --check --color always
```

Reply:
Dunno if this will actually get the action to show color output, but will be nice if it can!